sql/core/src/main/scala/org/apache/spark/sql/execution/window/WindowExecBase.scala

```diff
@@ -170,7 +170,7 @@ abstract class WindowExecBase(
               MutableProjection.create(expressions, schema))
         }
 
-        // Create the factory
+        // Create the factory to produce [[WindowFunctionFrame]]
```

Comment:
scaladoc links can't work here

Reply:
What do you think of this?
`// Create the factory to produce WindowFunctionFrame`

Reply:
Yes fine, just saying to remove the scaladoc brackets

Reply:
OK